docs/getting-started/create-comit-app.md

```diff
@@ -4,6 +4,312 @@ title: Getting Started
 sidebar_label: Create-comit-app
 ---
 
-We are working extensively on pushing out a new documentation. If you want to help out, feel free to grab a ticker from [here](https://github.com/comit-network/comit-network.github.io) :)
+[`create-comit-app`](https://github.com/comit-network/create-comit-app/) is a project to help COMIT App developers setup an environment.
```

Comment:
I think we should add a few more words here answering the question: "Why am I reading this doc?" 

Maybe follow the layout: 

# Introduction

[...]
# Summary/TL;DR - Quick Start

[...]
# Getting started 

[...]

Reply:
Hum, I just understood your proposal after doing the change locally.
I have added some explanation with 52fefe9b, let me know if it solves the issue.